ips_list.yml

```diff
@@ -75,6 +75,10 @@ ibex:
   commit: e65a27e14dda7bb3b09cadc4e9232305fd451cec
   group: lowRISC
   domain: [soc]
+cv32e40p:
+  commit: core-v-mcu
+  group: davideschiavone
```

Comment:
Is this a fork from you?

Reply:
Yes I explain in the PR message why. I need first this guy to be merged in the openHW group repository to use the official repo. Problem is that it will take time as we can't accept any longer PRs until we set a CI in place. So this is a way to go faster and do not block core-v-mcu. We can also do a branch  inside openhw is you prefer.

Reply:
I think a branch within OpenHW would be terrific. Just to avoid too many personal projects from growing into the OpenHW repos. What do you think?